install/kubernetes/cilium/values.yaml

```diff
@@ -2054,3 +2054,23 @@ enableK8sTerminatingEndpoint: true
 # -- Configure the key of the taint indicating that Cilium is not ready on the node.
 # When set to a value starting with `ignore-taint.cluster-autoscaler.kubernetes.io/`, the Cluster Autoscaler will ignore the taint on its decisions, allowing the cluster to scale up.
 agentNotReadyTaintKey: "node.cilium.io/agent-not-ready"
+
+dnsProxy:
```

Comment:
After #20066 was merged, now these changes need to be made to `install/kubernetes/cilium/values.yaml.tmpl` instead, then run `make -C install/kubernetes` to update this file.